### DISCUSSION TITLE
How do I specify the build parameter ?

### DISCUSSION THREAD

**Discussion by goyzhang** [stryker-mutator/stryker-net#1661] [Q&A] [answered]
For example, I probably want to do `MSbuild XXX.sln -p:Configuration=Release;Platform=Platform1;CustomProp=ABC`. 
Is there any way to specify the build parameter?

**Comment by rouke-broersma** [comment, marked as answer]
There currently is no way to set these except using msbuild properties in the csproj or perhaps environment variables.

**Reply by goyzhang** [reply]
OK, that's unfortunate, my project has different releases by specifying different compile-time properties. I am able to do it via `dotnet test/build -p` so I wonder if possible for stryker since it internally uses `dotnet`?

**Reply by rouke-broersma** [reply]
Theoretically everything is possible. But we have not built this functionality so no currently you can only do this with msbuild conditions and environment variables.
